src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -48,6 +48,7 @@
 					class="tw-mt-3"
 					:class="{ 'tw-order-last' : enableRelendingExp }"
 					:enable-five-dollars-notes="enableFiveDollarsNotes"
+					:enable-qf-mobile="enableQFMobileVersion"
```

Comment:
I see we're reusing an old experiment, do we want to update the analytics label:

https://github.com/kiva/ui/blob/cc1ffa5a0ebb89d1d5ddd362d8be61eb8e430fb4/src/pages/LoanFinding/LoanFinding.vue#L415